scout/models/variant/variant.py

```diff
@@ -46,8 +46,16 @@
     dbsnp_id=str,
     # str variant
     str_ru=str,
+    str_display_ru=str,
```

Comment:
what does _ru stand for?

Reply:
Repeat Unit - those are not very self explanatory but same as in Stranger and ExHu.